rdflib/query.py

```diff
@@ -218,7 +218,7 @@ def serialize(
         self,
         destination: Optional[Union[str, IO]] = None,
         encoding: str = "utf-8",
-        format: str = "xml",
+        format: str = "ttl",
```

Comment:
This is breaking our interface so it will have to wait for 7.0, for more info see this discussion:
- https://github.com/RDFLib/rdflib/discussions/1841